rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -842,23 +842,25 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
     }
 
     /**
-     * calls {@link #saveExecutionState_currentTransaction(java.lang.Object, java.lang.Object, java.util.Map, rundeck.services.ExecutionService.AsyncStarted, java.util.Map)}
+     * calls {@link #saveCompletedExecution_currentTransaction(java.lang.Object, java.lang.Object, java.util.Map, rundeck.services.ExecutionService.AsyncStarted, java.util.Map)}
      * @param e execution
      * @param status
      */
     private void cleanupExecution_currentTransaction(Execution e, String status = null) {
```

Comment:
The `cleanupExecution_currentTransaction` function name is a little strange now that we are changing the code to create inner transactions. Once we do this it essentially becomes the same as the other `cleanupRunningJobs` methods in this service. 

What do you think about the following:

* Delete this function.
* Delete [the only caller of this function](https://github.com/rundeck/rundeck/blob/a7dfef49f6c6c89f8747289b42e4ca3587425cf5/rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy#L821-L827) and [the only caller of that function](https://github.com/rundeck/rundeck/blob/a7dfef49f6c6c89f8747289b42e4ca3587425cf5/rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy#L798-L804)
* Change the code in `rundeckpro` to instead call [the other cleanupRunningJobs method](https://github.com/rundeck/rundeck/blob/a7dfef49f6c6c89f8747289b42e4ca3587425cf5/rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy#L795-L797).